From df062a52f093bd36c81e866a1afa7239e6adbee0 Mon Sep 17 00:00:00 2001
From: Tom Siewert <tom.siewert@hetzner.com>
Date: Wed, 11 Jan 2023 07:03:03 +0100
Subject: [PATCH] [Makefile] Get echo path via env

Some linux distributions (e.g. NixOS) do not provide "echo" in /bin.
Using /usr/bin/env to get the path of echo fixes this issue.

Signed-off-by: Tom Siewert <tom.siewert@hetzner.com>
---
 src/Makefile.housekeeping | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/Makefile.housekeeping b/src/Makefile.housekeeping
index 9bf34b2e..7161cca7 100644
--- a/src/Makefile.housekeeping
+++ b/src/Makefile.housekeeping
@@ -10,8 +10,8 @@
 TAB 			:= $(shell $(PRINTF) '\t')
 ECHO_E_ECHO		:= $(ECHO)
 ECHO_E_ECHO_E		:= $(ECHO) -e
-ECHO_E_BIN_ECHO 	:= /bin/echo
-ECHO_E_BIN_ECHO_E 	:= /bin/echo -e
+ECHO_E_BIN_ECHO 	:= /usr/bin/env echo
+ECHO_E_BIN_ECHO_E 	:= /usr/bin/env echo -e
 ECHO_E_ECHO_TAB		:= $(shell $(ECHO_E_ECHO) '\t' | cat)
 ECHO_E_ECHO_E_TAB	:= $(shell $(ECHO_E_ECHO_E) '\t' | cat)
 ECHO_E_BIN_ECHO_TAB 	:= $(shell $(ECHO_E_BIN_ECHO) '\t')
-- 
2.38.1

